.github/workflows/create_containers.yml

```
@@ -11,31 +11,25 @@ jobs:
   push_to_registry:
     permissions:
       packages: write # for docker/build-push-action
-    runs-on: ${{ matrix.runner }}
+    runs-on: ${{ case(matrix.ARCH == "arm64", "ubuntu-24.04-arm", "ubuntu-latest") }}
```

actionlint (1.7.12)

[error] got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter

(expression)